### PR TITLE
Resolve openstack-exporter#135

### PR DESCRIPTION
**Pull request by shotgun-20**
Ironic exporter now checks for API microversion. Also, baremetal resource_class metric is added. Release tarballs added to gitignore.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.2.0
+1.2.1
```
Comment (niedbalski) [inline VERSION:1]: There is no need to increase the version number, please revert this change.

**Comment by niedbalski** [comment]
@shotgun-20 hello, I am awaiting the tests to come back and please check the comment made on the VERSION file.

File: VERSION (hunk 1 above)
Comment (shotgun-20) [reply VERSION]: Done, reverted it.

**Comment by shotgun-20** [comment]
Done, no version change here anymore.

**Comment by niedbalski** [comment]
LGTM @shotgun-20 ,i'll wait for tests and merge. Once again thank you for your contributions!
